common/predictive-text/package.json

```diff
@@ -1,25 +1,25 @@
 {
-  "name": "keyman-lmlayer-prototype",
-  "version": "0.0.0",
+  "name": "@keymanapp/language-modeling-layer",
```

Comment:
Still hoping for `@keymanapp/lexical-model-layer` in order to match with the types and compiler names etc.

Reply:
I'm going to need some convincing on that.  Thoughts, @eddieantonio?

My perspective was that lexical models are a _part of_ the lm-layer, not the entirety.  During our original design process, we noted the potential for other types of language-models to be involved in the process.  "lexical models" are a strict subset of "language models".  If we start involving any sort of contextual model, say... bi-grams or n-grams... those wouldn't be lexical models, but instead a separate 'layer' of language modeling.

Reply:
I don't think I was as clear on the distinction in my mind as you seem to be!

If that's the case, the lexical model compiler needs to be renamed to language-model-compiler. Because the majority of what it does is not specific to lexical models. Conceptually, they are paired. And I don't actually think we'd be developing a whole new set of types for other types of language models: the type data in `@keymanapp/lexical-model-types` is generic to any language model, I think.

Reply:
I was kind of thinking the same thing after I posted.  The one issue there - we actually do [publish that module](https://www.npmjs.com/package/@keymanapp/lexical-model-types).

I know there are ways to rename stuff within npm, so it can be fixed longterm once we're ready to do so.  But that'd take a bit of time and investigation, as I don't know the actual specifics yet... and that may break things for some people.  So... hmm.

Reply:
Haha, I think we've named ourselves into a corner. For internal consistency, I vote for `@keymanapp/lexical-model-layer`, though I agree with Joshua that it is the _language modelling_ layer; not really the "lexical model" layer ¯\\\_(ツ)\_/¯